src/components/CustomAnimatedFlatList/index.tsx

```diff
@@ -0,0 +1,121 @@
+import type {PropsWithChildren} from 'react';
```

Comment:
Could you leave a comment here, that this is taken from `react-native-reanimated` and that this should be updated when there's an update to the original implementation in the library